packages/grid/src/vaadin-grid-array-data-provider-mixin.js

```diff
@@ -22,7 +22,7 @@ export const ArrayDataProviderMixin = (superClass) =>
     }
 
     static get observers() {
-      return ['__dataProviderOrItemsChanged(dataProvider, items, isAttached, items.*, _filters, _sorters)'];
+      return ['__dataProviderOrItemsChanged(dataProvider, items, isAttached, _filters, _sorters, items.*)'];
```

Comment:
Was is intentional that `items.*` just moved to the end of the argument list?

Reply:
Yeah, it was restored in https://github.com/vaadin/web-components/pull/6677/commits/d54fd5ce3dd49a0ae185ef5b36ff0d69e71b5cf4 since an integration test which uses `<template>` failed.

I imagined that since it's kind of an "extra" dependency, to be removed later (once the template adapter is removed in V25), it's nicer to have it as the last one on the list.